bundles/org.eclipse.rap.tools.launch.rwt/src/org/eclipse/rap/tools/launch/rwt/internal/delegate/RWTLaunchDelegate.java

```diff
@@ -81,7 +81,7 @@ public String[] getClasspath( ILaunchConfiguration configuration ) throws CoreEx
     list.add( BundleFileLocator.locate( "org.eclipse.jetty.util.ajax" ) ); //$NON-NLS-1$
     list.add( BundleFileLocator.locate( "org.eclipse.jetty.webapp" ) ); //$NON-NLS-1$
     list.add( BundleFileLocator.locate( "org.eclipse.jetty.xml" ) ); //$NON-NLS-1$
-    list.add( BundleFileLocator.locate( "org.slf4j.api" ) ); //$NON-NLS-1$
+    list.add( BundleFileLocator.locate( "slf4j.api" ) ); //$NON-NLS-1$
```

Comment:
@ifurnadjiev Although the tests are still green (after removing `slf4j.api` from the list of bundles in the corresponding test case), I have my doubts whether this is already working / sufficient. Maybe we can talk about it.

Reply:
Leaving it for now. Unless we find out problems during our manual tests, it is probably a false finding.